invokeai/frontend/web/src/features/stylePresets/components/StylePresetMenuTrigger.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, Icon } from '@invoke-ai/ui-library';
+import { useAppDispatch, useAppSelector } from 'app/store/storeHooks';
+import { isMenuOpenChanged } from 'features/stylePresets/store/stylePresetSlice';
+import { useCallback } from 'react';
+import { PiCaretDownBold } from 'react-icons/pi';
+
+import { ActiveStylePreset } from './ActiveStylePreset';
+
+export const StylePresetMenuTrigger = () => {
+  const isMenuOpen = useAppSelector((s) => s.stylePreset.isMenuOpen);
+  const activeStylePreset = useAppSelector((s) => s.stylePreset.activeStylePreset);
+  const dispatch = useAppDispatch();
+
+  const handleToggle = useCallback(() => {
+    dispatch(isMenuOpenChanged(!isMenuOpen));
+  }, [dispatch, isMenuOpen]);
+
+  return (
+    <Flex
+      as="button"
+      onClick={handleToggle}
+      backgroundColor="base.800"
+      justifyContent="space-between"
+      alignItems="center"
+      padding="5px 10px"
+      borderRadius="base"
+      gap="2"
+      borderTop="2px solid transparent"
+      borderColor={activeStylePreset ? 'invokeBlue.200' : 'transparent'}
+    >
+      <ActiveStylePreset />
+
+      <Icon boxSize="15px" as={PiCaretDownBold} color="base.300" />
```

Comment:
Please do not use pixel values for stuff like boxSize, padding, gap unless necessary. Use the theme tokens (e.g. numbers without units). When pixel values are used it means we have to manually update all the values if we ever change the theme.

This icon needs a bit of lefthand padding to look right.